www/README.md

```diff
@@ -2,6 +2,12 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+## Authentication set-up
+
+1. Copy `www/.env.sample` to `www/.env`
+1. Navigate to `compe-plus` application in ![Auth0](https://manage.auth0.com/dashboard)
```

Comment:
Who has the account on auth0?

Reply:
The compe club account will. I still need to get in touch with Cyrus to get the credentials

Reply:
Okay